Source/WebCore/rendering/LayoutRepainter.h

```diff
@@ -48,6 +50,7 @@ class LayoutRepainter {
     LayoutRect m_oldBounds;
     LayoutRect m_oldOutlineBox;
     bool m_checkForRepaint;
+    bool m_repaintOutlineBox;
```

Comment:
Can we call this `m_repaintOutlineBounds ` for consistency with `outlineBoundsForRepaint()`?